src/main/java/frc/robot/RobotContainer.java

```diff
@@ -105,7 +114,43 @@ public void burnFlashSparks() {
    * joysticks}.
    */
   private void configureBindings() {
-    driverController.a().whileTrue(new InstantCommand(intake::deploy, intake));
+    driverController.a().onTrue(new InstantCommand(drive::deploySkids, drive));
+    driverController.b().onTrue(new InstantCommand(lift::getRidOfObject, lift));
+    driverController.x().onTrue(new InstantCommand(lift::cancelScore, lift));
+    driverController.y().whileTrue(new InstantCommand(lift::manualScore, lift));
+
+    driverController.back().onTrue(new InstantCommand(lift::home, lift));
+    driverController.start().onTrue(new InstantCommand(drive::halfSpeedToggle, drive));
+
+    // TODO Maybe: steal
+    driverController.rightBumper().whileTrue(new InstantCommand(lift::prepScore, lift));
```

Comment:
This should probably be a press, also this will be a fully fledged command but the stub function is fine for now